src/spatialdata/_io/format.py

```diff
@@ -69,12 +100,18 @@ def validate_coordinate_transformations(
 
             assert np.all([j0 == j1 for j0, j1 in zip(json0, json1)])
 
+    # eventually we are fully compliant with NGFF and we can drop SPATIALDATA_FORMAT_VERSION and simply rely on
+    # "version"; still, until the coordinate transformations make it into NGFF, we need to have our extension
+    @property
+    def spatialdata_format_version(self) -> str:
+        return "0.1"
+
 
-class ShapesFormatV01(SpatialDataFormatV01):
+class ShapesFormatV01(SpatialDataFormat):
     """Formatter for shapes."""
 
     @property
-    def version(self) -> str:
+    def spatialdata_format_version(self) -> str:
```

Comment:
I don't undersatnd the renaming to spatialdata_format_version, if it's really just the version of the element?

Reply:
The problem was that the various format classes (`PointsFormatV01`, ...) are inheriting from `ome_zarr.format.CurrentFormat`, so the property `format` was overriding the one from `ome_zarr`. This is ok for points, shapes and tables, but not for images and labels. In fact, since we use `ome_zarr` for some parts of IO for images and labels, we need the original `format`. But we need also the spatialdata `format`. So I thought that renaming the spatialdata format to not have this clash is cleaner.

Reply:
I think it's better to have these two property separate.